File: project01/aditya-mishra-project01.py
#2A. In Python

import socket
print(socket.gethostname())

#3.

import pandas as pd
df = pd.read_csv('/anvil/projects/tdm/data/flights/subset/1991.csv')
df[df["Month"]==12].head() # see information about a few of the flights from December 1991

#4A. In Python

mem = 256000 #Memory in Anvil subcluster A in GB
gb_to_b = 1000000000 #Conversion factor from GB to Bytes
gb_to_tb = 1000 #Conversion factor from GB to TB

memb = str(mem * gb_to_b)
memtb = str(mem / gb_to_tb)

print("The memory of Anvil subcluster A is " + memb + " bytes.")
print("The memory of Anvil subcluster A is " + memtb + " terabytes.")

File: project01/aditya-mishra-project01.R
#2C. In R

system("hostname", intern=TRUE)

#4B. In R

mem <- 256000 #Memory in Anvil subcluster A in GB
gb_to_b <- 1000000000 #Conversion factor from GB to Bytes
gb_to_tb <- 1000 #Conversion factor from GB to TB

memb <- mem * gb_to_b
memtb <- mem / gb_to_tb

print(paste("The memory of Anvil subcluster A is", memb, "bytes."))
print(paste("The memory of Anvil subcluster A is", memtb, "terabytes."))

#5.

dat <- read.csv("/anvil/projects/tdm/data/disney/flight_of_passage.csv")
head(dat)

flight_of_passage <- dat

rm(dat)

head(flight_of_passage)